.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -1345,10 +1353,11 @@ public enum ChatMessageContentPartKind {
     }
     public enum ChatMessageRole {
         System = 0,
-        User = 1,
-        Assistant = 2,
-        Tool = 3,
-        Function = 4
+        Developer = 1,
```

Comment:
Is it a problem that this changed the int values? It generated this way, but we could always customize to force the int values to remain stable (add `Developer` as `5`)

Reply:
Yes, you're correct. We should customize it to keep the values to remain stable.

Reply:
Addressed!